pbs/pbsrequest.go

```diff
@@ -127,6 +135,17 @@ func ParsePBSRequest(r *http.Request, cache cache.Cache) (*PBSRequest, error) {
 	if pbsReq.Device == nil {
 		pbsReq.Device = &openrtb.Device{}
 	}
+	if pbsReq.SDK == nil {
+		pbsReq.SDK = &SDK{}
+	}
+	if pbsReq.SDK.Version != "0.0.1" {
+		if pbsReq.PBSUser != nil {
```

Comment:
My concern with this is that it's specific to the AN adapter, where the PBS request is applicable to all adapters. Is there a way to move this logic into AN adapter?